Automatic Validation ended with:
> 2024-07-23T14:49:02.9699761Z Manifest Validation Failed
 2024-07-23T14:49:08.0269298Z ##[error]Manifest Validation Failed

(Automated response - build 887.)

--- a/manifests/h/Headlamp/Headlamp/0.24.1/Headlamp.Headlamp.installer.yaml
+++ b/manifests/h/Headlamp/Headlamp/0.24.1/Headlamp.Headlamp.installer.yaml
@@ -1,0 +1,17 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Headlamp.Headlamp
+PackageVersion: 0.24.1
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: user
+InstallModes:
+- interactive
+- silent
+ReleaseDate: 2024-06-27
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/headlamp-k8s/headlamp/releases/download/v0.24.1/Headlamp-0.24.1-win-x64.exe
+  InstallerSha256: b52d4cfacb991be848864c8fc9b0e388452be1ef003113ff48b25c292ca72fc1
+ManifestType: installer
+ManifestVersion: 1.6.0

--- a/manifests/h/Headlamp/Headlamp/0.24.1/Headlamp.Headlamp.locale.en-US.yaml
+++ b/manifests/h/Headlamp/Headlamp/0.24.1/Headlamp.Headlamp.locale.en-US.yaml
@@ -1,0 +1,47 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Headlamp.Headlamp
+PackageVersion: 0.24.1
+PackageLocale: en-US
+Publisher: Headlamp
+PublisherUrl: https://headlamp.dev/
+PackageName: Headlamp
+License: Apache-2.0
+LicenseUrl: https://github.com/headlamp-k8s/headlamp/blob/HEAD/LICENSE
+ShortDescription: Kubernetes web UI.
+Description: A Kubernetes web UI that is fully-featured, user-friendly and extensible.
+Tags:
+- cncf
+- dashboard
+- headlamp
+- kinvolk
+- kubernetes
+- logging
+- monitoring
+- plugins
+- ui
+- web
+ReleaseNotes: |-
+  🐞 Bug fixes
+  - Fix viewing the Network > Services page when there are services without ports
+  - Fix top bar and sidebar on small screens (thanks to @tazo90 )
+  - Fix misc regressions in several components
+  - Do not log errors related to OIDC when OIDC is not set up
+  - Remove mistakenly added package*.json files from root folder
+  - Fix hiding back link when it's set as null
+  - charts: Fix wrong OIDC external secret loading
+  - charts: Fix config.extraArgs and env values type
+  - charts: Fix ingress (thanks to @BlindB0 )
+  💻 Development
+  - chart: Add schema for values
+  - Make KubeService.spec.ports as optional in interface
+  - Update some dependencies
+  📖 Documentation
+  - Fix link to FAQ in README
+  - chart: Add example docs for ingress
+ReleaseNotesUrl: https://github.com/headlamp-k8s/headlamp/releases/tag/v0.24.1
+Documentations:
+- DocumentLabel: headlamp documentation
+  DocumentUrl: https://headlamp.dev/docs/latest
+ManifestType: defaultLocale
+ManifestVersion: 1.6.0

--- a/manifests/h/Headlamp/Headlamp/0.24.1/Headlamp.Headlamp.yaml
+++ b/manifests/h/Headlamp/Headlamp/0.24.1/Headlamp.Headlamp.yaml
@@ -1,0 +1,5 @@
+PackageIdentifier: Headlamp.Headlamp
+PackageVersion: 0.24.1
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.6.0

